source/layouts/components/_reference_table_of_contents.haml

```diff
@@ -1,4 +1,4 @@
-%nav.sl-c-list-navigation-wrapper.sl-c-list-navigation-wrapper--collapsible
-  %h3 Contents
+%div.sl-c-list-navigation-wrapper.sl-c-list-navigation-wrapper--collapsible
```

Comment:
In haml, you don't need to define a `div` — the classes will automatically be a `div` when there is no element defined. :)

Reply:
Same here.